extensions/cornerstone/src/commandsModule.ts

```diff
@@ -770,8 +753,8 @@ function commandsModule({
     toggleReferenceLines: {
       commandFn: actions.toggleReferenceLines,
     },
-    storePresentation: {
-      commandFn: actions.storePresentation,
+    resetCameraLayout: {
```

Comment:
Hmm... `storePresentation` has been moved/removed, yet I still see commands being run for it. I believe those were added for the PR #3560. I believe that functionality is broken now.

Reply:
yeah I didn't meant to commit it